### PR TITLE
Event cards

### LINKED ISSUE #5: Upcoming Events Display
As a user, when I load the page, I should see:
- [x] The main section of the page with upcoming events displayed (may need to adjust to load in chunks depending on how BE is set up)

### PR DESCRIPTION
**Pull request by megan-venetianer**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Styling- no new features
- [ ] Refactoring- no new features
- [ ] Testing

### Description:
This PR adds the EventsContainer and EventCard components and adds the styling for each of the rendered event cards.

### Related Issues/Tickets:
Closes #5 

### Files modified:
- public/empty-heart.png
- public/filled-heart.png
- src/App.vue
- src/components/EventCard.vue
- src/components/EventsContainer.vue
- src/components/Nav.vue

### Test Results:
N/A

### Notes:

### CODE REVIEW

**Comment by CLPena** [comment]
Woohoo! We made some good progress today. Just going to document a few notes here for the sake of keeping a paper trail:

- We still need conditionals to fill in the image or the default "no image available", depending on data received
- There are currently five cards hard-coded into this, which we did for the sake of styling. Once we have data to work with, we will make them dynamically load.
- We need to eventually change the color of the favorite button based on state
- Once we have data, we will want to double-check that the positioning for everything still looks good. Might have to add a max-height and scroll feature for event names (or some kind of cut-off with max characters)

**Comment by megan-venetianer** [comment]
Thanks for documenting all of those additional to-dos! Great idea and much appreciated!

File: src/App.vue [hunk 1]
```diff
@@ -45,6 +37,14 @@ export default {
     font-family: 'Roboto Condensed', sans-serif;
   }
 
+  #app {
```
Comment (dreardon1021) [inline src/App.vue:40]: Whats the plan for styling the apps vh when content extends 100vh. 

min-vh might be better option here

File: src/components/Nav.vue [hunk 2]
```diff
@@ -32,7 +32,6 @@ export default {
        background: #118AB2;
        height: 100vh;
        width: 22%;
-       position: sticky;
        display: flex;
        flex-flow: column nowrap;
```
Comment (dreardon1021) [inline src/components/Nav.vue:36]: nowrap is default.
flex-direction: column;  ---> is all you need

File: src/components/EventsContainer.vue [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+<style scoped>
+
+h1 {
+  color: #EF476F;
+  font-size: 3.5em;
+  margin-top: .18em;
+}
+
+.events-container {
```
Comment (dreardon1021) [inline src/components/EventsContainer.vue:32]: min-vh might apply here too

File: src/App.vue (hunk 1 above)
Comment (CLPena) [reply src/App.vue:40]: We have a card on the project board to decide on this when we have actual data to work with and can try out different options. We've discussed an infinite scroll situation or pagination, but we're open to other ideas!

Comment (CLPena) [reply src/App.vue:40]: But yes, good call on min-vh being an option!

Comment (dreardon1021) [reply src/App.vue:40]: Coolio, merging

File: src/components/Nav.vue (hunk 2 above)
Comment (CLPena) [reply src/components/Nav.vue:36]: I think we did it this way for when we eventually add a breakpoint for mobile. That way, we can easily switch it to a header at the top of the page just by changing the flex-flow property. But happy to change it to flex-direction if you think that's better!

**Pull request merged**